Classes/People/PeopleViewController.swift

```diff
@@ -0,0 +1,134 @@
+    private var selections = Set<String>()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.refreshControl = feedRefresh.refreshControl
+        feedRefresh.refreshControl.addTarget(self, action: #selector(LabelsViewController.onRefresh), for: .valueChanged)
+
+        feedRefresh.beginRefreshing()
+        fetch()
+        updateSelectionCount()
+    }
+
+    // MARK: Private API
+
+    func updateSelectionCount() {
+        let label = UILabel()
+        label.font = Styles.Fonts.body
+        label.backgroundColor = .clear
+        label.textColor = Styles.Colors.Gray.light.color
+        label.text = "\(selections.count)/\(selectionLimit)"
+        label.sizeToFit()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: label)
+    }
+
+    @objc func onRefresh() {
+        fetch()
+    }
+
+    func fetch() {
+        client.fetchAssignees(owner: owner, repo: repo) { [weak self] result in
+            self?.feedRefresh.endRefreshing()
+            switch result {
+            case .success(let users):
+                self?.users = users
+                self?.tableView.reloadData()
+            case .error:
+                ToastManager.showGenericError()
+            }
+        }
+    }
+
+    @IBAction func onDone(_ sender: Any) {
+        let selections = users.filter { self.selections.contains($0.login) }
+        delegate?.didDismiss(controller: self, selections: selections)
+        dismiss(animated: true)
+    }
+
+    // MARK: Public API
+
+    func configure(
+        selections: [String],
+        type: PeopleType,
+        client: GithubClient,
+        delegate: PeopleViewControllerDelegate,
+        owner: String,
+        repo: String
+        ) {
+        self.selections = Set<String>(selections)
+        self.type = type
+        self.client = client
+        self.delegate = delegate
+        self.owner = owner
+        self.repo = repo
+
+        switch type {
+        case .assignee: title = NSLocalizedString("Assignees", comment: "")
+        case .reviewer: title = NSLocalizedString("Reviewers", comment: "")
+        }
+    }
+
+    // MARK: UITableViewDataSource
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return users.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        let user = users[indexPath.row]
+        if let cell = cell as? PeopleCell, let avatarURL = URL(string: user.avatar_url) {
+            let login = user.login
+            cell.configure(avatarURL: avatarURL, username: login, showCheckmark: selections.contains(login))
+        }
+        return cell
+    }
+
+    // MARK: UITableViewDelegate
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```

Comment:
https://developer.apple.com/documentation/uikit/uitableviewdelegate/1614943-tableview

We could implement `willSelectRowAt` and return `nil` if the user would exceed their selection count (preventing the visual selecting of the cell?

Another alternative would be to do a quick error modal saying "Exceeded Selection Limit"?

Reply:
I like the `willSelectRowAt` idea, that'll be clean. The error message might be a bit much, hoping people can pick up on the "3/10" label in the top left. Maybe we can do a scale "pop" animation on that label to draw their attention? IMO details we can add later.